commands/serve/lib/webpack.build.js

```diff
@@ -56,6 +56,7 @@ const cfg = ({ srcDir, distDir, dev = false, serveConfig = {} }) => {
         {
           enforce: 'pre',
           test: /\.js?$/,
+          exclude: [/node_modules[/\\]@qlik[/\\]sdk/],
```

Comment:
I don't get why this is needed? What makes the sdk different from all other dependencies?

Reply:
there is a problem with source map of `@qlik/sdk` which cause 13 warnings in terminal. So as we discussed it in [here](https://qlikdev.slack.com/archives/C01BX56DX1C/p1663585679777969) with Tobias Linsefors. seems like excluding sdk's source map is an ok solution for it.

Reply:
There is something different (wrong?) with the source map in @qlik/sdk that makes the source-map-loader plugin fail to understand it.

Other options cloud be to use [filterSourceMappingUrl](https://webpack.js.org/loaders/source-map-loader/#filtersourcemappingurl) like in sense-client
Or maybe disable the warning like this: https://webpack.js.org/loaders/source-map-loader/#filtersourcemappingurl

I think this is OK.
But after reading the (source-map-loader) docs I think [disable the warning](https://webpack.js.org/loaders/source-map-loader/#filtersourcemappingurl) (if it works) is a better solution. As if the @qlik/sdk source map is fixed it would start using it without any change needed.

Reply:
Good Idea, 
✅ `ignoreWarnings` property added to webpack config file!